ExampleGallery/Infrastructure/CustomThumbnails.cs

```diff
@@ -59,26 +60,26 @@ IRandomAccessStream ICustomThumbnailSource.Thumbnail
         }
     }
 
-
+    // vvvvvvvv the below method is commented out because CustomControlExample is not supported in WinUI3 yet.
```

Comment:
vvvvvvv

Reply:
those are supposed to be down arrows but that was probably a misstep